Training_analysis.py

```diff
@@ -110,11 +123,24 @@
 #     
 
 # +
-model_logs_root = 'logs/sample-console-logs'
-log = DeepRacerLog(model_logs_root)
+# Specify your bucket name and prefix to load S3 logs, prefix should not including a'/' at the start or the end
+PREFIX='Demo-Reinvent'
+BUCKET='deepracer-local'
+fh = S3FileHandler(bucket=BUCKET,prefix=PREFIX)
+
+#Specify if using multiple workers to true, otherwise set to false
+MULTIPLE_WORKERS='true'
```

Comment:
Why using string 'true', rather than boolean True?

Reply:
It would also be possible to detect this if one sees more than one worker in the log.

Reply:
updated it so it now autodetects multiple workers using same test as in the underlying utils python code: -

 if df.nunique(axis=0)['worker'] > 1: